test/Common/DurableOrchestrationClientMock.cs

```diff
@@ -7,9 +7,9 @@
 
 namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
 {
-    internal class DurableOrchestrationClientMock : DurableOrchestrationClient, IDurableOrchestrationClient
+    internal class DurableOrchestrationClientMock : DurableClient, IDurableOrchestrationClient
```

Comment:
NIT: I know that this isn't a change you made, but I am not sure why this also needs to implement `IDurableOrchestrationClient`, as it should inherit it automatically from `DurableClient`.